app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardsSettingFragment.kt

```diff
@@ -46,6 +49,8 @@ class CreditCardsSettingFragment : PreferenceFragmentCompat() {
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         setPreferencesFromResource(R.xml.credit_cards_preferences, rootKey)
+
+        setupAutofillPreference()
```

Comment:
Maybe we should just inline this since the code is fairly straight forward.

Reply:
Sure.